doc/source/cluster/kubernetes/user-guides/gke-gcs-bucket.md

```diff
@@ -0,0 +1,168 @@
+# Configuring KubeRay to use Google Cloud Storage Buckets in GKE
+
+If you are already familiar with Workload Identity in GKE, you can skip this document.  The gist is that you need to specify a service account in each of the Ray pods after linking your Kubernetes service account to your Google Cloud service account.  Otherwise, read on.
+
+We will follow an abridged version of the documentation at <https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity>.  The full documentation is worth reading if you are interested in the details.
+
+## Create a Kubernetes cluster on GKE
+
+For this example, we will create a minimal KubeRay cluster using GKE.
+
+Run this command and all following commands on your local machine or on the [Google Cloud Shell](https://cloud.google.com/shell). If running from your local machine, you will need to install the [Google Cloud SDK](https://cloud.google.com/sdk/docs/install).
+
+```bash
+gcloud container clusters create cloud-bucket-cluster \
+    --num-nodes=1 --min-nodes 0 --max-nodes 1 --enable-autoscaling \
+    --zone=us-west1-b --machine-type e2-standard-8 \
+    --workload-pool=my-project-id.svc.id.goog # Replace my-project-id with your GCP project ID
+```
+
+
+This command creates a Kubernetes cluster named `cloud-bucket-cluster` with 1 node in the `us-west1-b` zone. In this example, we use the `e2-standard-8` machine type, which has 8 vCPUs and 32 GB RAM.
+
+Now get credentials for the cluster to use with `kubectl`:
+
+```bash
+gcloud container clusters get-credentials cloud-bucket-cluster --zone us-west1-b --project my-project-id
+```
+
+## Create an IAM Service Account
+
+```bash
+gcloud iam service-accounts create my-iam-sa
+```
+
+## Create a Kubernetes Service Account
+
+```bash
+kubectl create serviceaccount my-ksa
+```
+
+## Link the Kubernetes Service Account to the IAM Service Account and vice versa
+
+In the following two commands, replace `default` with your namespace if you are not using the default namespace.
+
+```bash
+gcloud iam service-accounts add-iam-policy-binding my-iam-sa@my-project-id.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
```

Comment:
Where is `roles/iam.workloadIdentityUser` from?

Reply:
I'm not sure about the details. I think it's a predefined role built into GCP.  The user can find more details at https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity which is linked in the doc